modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
```

Comment:
Missing assembly comment

Reply:
I mention the assembly [on line 3](https://github.com/openshift/openshift-docs/pull/62775/files#diff-d4fdf1eb9fd7961c7ed561411244b04dffd05ed2ab84fb2fce8b85b78770871dR3). Or are you referring to something else?

Reply:
oop nevermind :)